mattermost-plugin/product/api_adapter.go

```diff
@@ -131,31 +135,23 @@ func (a *serviceAPIAdapter) HasPermissionToChannel(askingUserID string, channelI
 	return a.api.permissionsService.HasPermissionToChannel(askingUserID, channelID, permission)
 }
 
-//
 // Bot service.
-//
```

Comment:
:cry:  Removing the empty comment lines associates the comment directly with the method following when viewing docs in the IDE or auto-generated docs.  These comment are for demarcation of API groups.

Reply:
Reverted change, had to add a blank line to prevent from auto-removing.